### PR TITLE
OCP-14985: re-create the node after deletion

### PR DESCRIPTION
**Pull request by rbbratta**
With openshift/verification-tests#626 we can
save the node yaml and use it to re-create
the node after deletion

### CODE REVIEW

File: features/step_definitions/node.rb [hunk 1]
```diff
@@ -591,3 +591,33 @@
   nodes = BushSlicer::Node.list(user: admin)
   cb[cb_name] = nodes.select { |n| n.is_worker? }
 end
+
+Given /^I store the node #{QUOTED} to the#{OPT_SYM} clipboard$/ do |node, cb_name|
+  cb_name ||= :deleted_node
+  @result = admin.cli_exec(:get, resource: 'node', resource_name: node, o: 'yaml')
```
Comment (liangxia) [inline features/step_definitions/node.rb]: Add `ensure_admin_tagged` as admin is used.

**Comment by zhaozhanqi** [comment]
@rbbratta thanks the updating. could you also post the debug logs here?

**Comment by rbbratta** [comment]
too big for pastebin.  Try here:  https://gitlab.cee.redhat.com/snippets/1792

**Comment by zhaozhanqi** [comment]
/lgtm
you can use https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3 to debug but maybe many CI job is occupy the resource.

**Comment by rbbratta** [comment]
yeah, I tried Runner-v3 but it was clogged up, so I ran it locally.

**Comment by rbbratta** [comment]
rebased

**Pull request merged**